internal/vendors/supermicro/supermicro.go

```diff
@@ -271,6 +276,64 @@ func (ob *outBand) Console(s ssh.Session) error {
 	return ob.IpmiTool.OpenConsole(s)
 }
 
+func (ob *outBand) UpdateBIOS(board, revision string, s3Config *api.S3Config) error {
+	update, err := ob.downloadFirmwareUpdate("bios", board, revision, s3Config)
+	if err != nil {
+		return err
+	}
+
+	err = ob.PowerOff()
+	if err != nil {
+		return err
+	}
+	defer func() {
+		_ = ob.PowerOn()
+	}()
+
+	time.Sleep(time.Minute)
+
+	return ob.sum.UpdateBIOS(update)
+}
+
+func (ob *outBand) UpdateBMC(board, revision string, s3Config *api.S3Config) error {
+	update, err := ob.downloadFirmwareUpdate("bmc", board, revision, s3Config)
+	if err != nil {
+		return err
+	}
+
+	err = ob.PowerOff()
```

Comment:
Is it required to poweroff the machine to do a firmware update, i think no, it even does not need a powercycle at all. Please as @muhittink

Reply:
For a firmware update the machine needs to be powered off, but I will check again.
Real problem ist, that metal-core does not have sum tool.
Will fix this soon